### PR TITLE
Kp/link back to encore

### PR DESCRIPTION
**Pull request by ktp242**
This PR adds the link to search catalog with the same keywords that are in the search input field. It also adds the styles across different viewports. More than that, it adds some fix for the syntax.

I am not sure if anyone is using development instance, if no,  I can deploy it to development for testing tomorrow.

### CODE REVIEW

File: src/client/styles/main.scss [hunk 1]
```diff
@@ -121,11 +121,38 @@ header {
...
+
+    @include min-screen($desktop) {
+      font-size: 20px;
+      height: 48px;
+      padding: 0;
+    }
+
+    a {
+      position: relative;
+      top: 16px
```
Comment (ktp242) [inline src/client/styles/main.scss]: Hey @jackkim9! Thanks for fixing the styles for this PR. I think we miss a `;` (should be `top: 16px;`)  here so `npm start` can't bundle correctly now.

**Comment by ktp242** [comment]
Thanks! Works perfect now.

**Comment by ktp242** [comment]
Hi I think it should be another ticket because include the major layout changes.

May I know the reason why we want to make the adjustment? I just want to evaluate the task with the time we have before release.

I feel it will be a 2-3 point task.

CC @courtneymcgee @LucasCioffi

**Comment by jackkim9** [comment]
Sounds good. Thanks for your estimate.

Want to make sure if this link is easy to notice as some users will expect to see book results in this page.

**Pull request merged**